sdk-setup/src/mb2

```diff
@@ -2074,6 +2077,17 @@ run_package() {
     )
 }
 
+run_sign() {
+    [[ -e $STATEDIR/spec ]] || fatal "No previoud build found"
+    local retcode=0
+    rpms=$(find_rpms $all_opt "$OPT_OUTPUTDIR" "$STATEDIR/spec")
+    [[ $rpms ]] || fatal "No package found"
+    local method;
+    method="custom-rpm-sign"
+    $method "$@" ${rpms} 
```

Comment:
Are you planning to have an option to change the method? If not, then I suggest you just use custom-rpm-sign directly without the variable.

Reply:
There is no great need for this now. But it would be convenient for us to use our sign script.

Added a signature script as well